app/views/home/index.html.haml

```diff
@@ -8,7 +8,7 @@
   %p= t('.beta-label-text-3')
 %h3= t('.feedback')
 .block
-  %p= t('.feedback-text', feedbacklink: link_to(t('.links.feedback'), 'https://ukparliament-self.achieveservice.com/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-032e1b1d-958f-4402-9ab2-97a38ed9b899/AF-Stage9b7ecae8-0c85-4ee7-afda-b256cffe5f08/definition.json&redirectlink=%2F&cancelRedirectLink=%2F&category=AF-Category-48936b5c-9329-48c7-a0a5-563463aacadf', target: '_blank', title: 'web site opens in a new window')).html_safe
+  %p= t('.feedback-text', feedbacklink: link_to(t('.links.feedback'), 'https://ukparliament-self.achieveservice.com/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-032e1b1d-958f-4402-9ab2-97a38ed9b899/AF-Stage9b7ecae8-0c85-4ee7-afda-b256cffe5f08/definition.json&redirectlink=%2F&cancelRedirectLink=%2F&category=AF-Category-48936b5c-9329-48c7-a0a5-563463aacadf', target: '_blank', title: t('.links.new_window'))).html_safe
```

Comment:
Jaws and NVDA will read the anchor text and then the title attribute if present so it would essentially read 'feedback website opens in the new window'